app/javascript/src/QuickStarts/templates/basicAPIintegration.yaml

```diff
@@ -0,0 +1,200 @@
+metadata:
+  name: getting-started-with-quick-starts
+  # you can add additional metadata here
+  instructional: true
+spec:
+  displayName: Complete a basic API integration
+  durationMinutes: 20
+  # Optional type section, will display as a tile on the card
+  type: 
+    text: Quick start
+    # 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'grey'
+    color: grey
+  # - The icon defined as a base64 value. Example flow:
+  # 1. Find an .svg you want to use, like from here: https://www.patternfly.org/v4/guidelines/icons/#all-icons
+  # 2. Upload the file here and encode it (output format - plain text): https://base64.guru/converter/encode/image
+  # 3. compose - `icon: data:image/svg+xml;base64,<base64 string from step 2>`
+  # - If empty string (icon: ''), will use a default rocket icon
+  # - If set to null (icon: ~) will not show an icon
+  icon: data:image/svg+xml;base64,PCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjUxMiIgaGVpZ2h0PSI1MTIiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj4KPHRpdGxlPjwvdGl0bGU+CjxnIGlkPSJpY29tb29uLWlnbm9yZSI+CjwvZz4KPHBhdGggZD0iTTQ0OCA2NHY0MTZoLTMzNmMtMjYuNTEzIDAtNDgtMjEuNDktNDgtNDhzMjEuNDg3LTQ4IDQ4LTQ4aDMwNHYtMzg0aC0zMjBjLTM1LjE5OSAwLTY0IDI4LjgtNjQgNjR2Mzg0YzAgMzUuMiAyOC44MDEgNjQgNjQgNjRoMzg0di00NDhoLTMyeiI+PC9wYXRoPgo8cGF0aCBkPSJNMTEyLjAyOCA0MTZ2MGMtMC4wMDkgMC4wMDEtMC4wMTkgMC0wLjAyOCAwLTguODM2IDAtMTYgNy4xNjMtMTYgMTZzNy4xNjQgMTYgMTYgMTZjMC4wMDkgMCAwLjAxOS0wLjAwMSAwLjAyOC0wLjAwMXYwLjAwMWgzMDMuOTQ1di0zMmgtMzAzLjk0NXoiPjwvcGF0aD4KPC9zdmc+Cg==
+  description: |-
+    Create the required objects to complete a basic integration with 3scale.
+  introduction: |-
+    This quick start guides you to create the required objects to complete a basic integration with 3scale as an API provider including 7 tasks. 
+    
+    To learn more about API configuration, see the [3scale documentation](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.11/html/getting_started/first-steps-with-threescale_configuring-your-api#products-backends-for-threescale-apis_configuring-your-api).
+  tasks:
+    - title: Create a new backend
+      description: |-
+        A backend is an internal API that is bundled to a product. In this quick start, you first create a backend and then a product.In practice, you can create either product or backend first.
+        
+        ### Procedure
+
+        1. Go to _Backends_ page: click the context selector on the header and select <strong>Backends</strong>.
+        1. On the _Backends_ page, click <strong>Create backend</strong>.
+        1. On the _New backend_ page, fill in the details to create the backend. 
+        
+          Important fields:
+              - _System name_:  3scale needs this name to internally identify and refer to the backend.
+              - _Private Base URL_: It is the base URL endpoint of the private API.
```

Comment:
It might be better to have one line between No.40 and N0.41


<img width="405" alt="屏幕快照 2022-03-14 上午11 17 12" src="https://user-images.githubusercontent.com/5396309/158099023-ffda60de-9d95-4b68-892f-b3f58627372d.png">

Reply:
@josemigallas @deernano I'm pretty sure this is a css stylesheet solution. I'm not sure where to place the style and if it'll mess other things up because it's probably a global solution for nested unordered lists. Something like the following:

`ul.frames {
  font-size: 0;
}
ul.frames li {
  font-size: 14px; font-size:1.4rem;
  display: inline;
}`

Reply:
@dfennessy I see.  Let's just leave as what it is for now. Thx!